app/models/internal/impl/DefaultAlert.java

```diff
@@ -43,51 +40,21 @@ public UUID getId() {
         return _id;
     }
 
-    @Override
-    public Context getContext() {
-        return _context;
-    }
-
     @Override
     public String getName() {
         return _name;
     }
 
     @Override
-    public String getCluster() {
-        return _cluster;
-    }
-
-    @Override
-    public String getService() {
-        return _service;
-    }
-
-    @Override
-    public String getMetric() {
-        return _metric;
-    }
-
-    @Override
-    public String getStatistic() {
-        return _statistic;
+    public String getQuery() {
+        return _query;
     }
 
     @Override
     public Period getPeriod() {
         return _period;
     }
 
-    @Override
-    public Operator getOperator() {
-        return _operator;
-    }
-
-    @Override
-    public Quantity getValue() {
-        return _value;
-    }
-
     @Override
     public NagiosExtension getNagiosExtension() {
```

Comment:
Should we get rid of this?

Reply:
Are you commenting on the nagios extension part?  I think we should.